tap-client/src/components/home/HomeComponent.tsx

```diff
@@ -1,8 +1,126 @@
+import "./HomeComponent.css";
+import studentImg from "../../../src/assets/home-assets/stock-students.jpg"
+import planImg from "../../../src/assets/home-assets/stock-plan.jpg"
+import aiImg from "../../../src/assets/home-assets/stock-ai.jpg"
+import collegeImg from "../../../src/assets/home-assets/stock-TU.jpg"
```

Comment:
These images make the page look weird. I think you should use Lottie images and they're really simple to implement. Just download them from [here]( https://lottiefiles.com/search?q=students&category=animations) and  use the Lottie tag. Refer the MainFormComponent.tsx to see an example.

Reply:
Do you want the first TU image to be replaced as well? I think it looks good and that it's a good idea to open the page to explicit TU branding.

Reply:
If you want to use a TU image, I think you should search for a better one. The image you have isn't really matching well with the rest of page.